app/scripts/modules/amazon/src/loadBalancer/configure/AmazonLoadBalancerChoiceModal.tsx

```diff
@@ -133,6 +155,14 @@ export class AmazonLoadBalancerChoiceModal extends React.Component<
                   </div>
                 ))}
             </>
+            {Boolean(loadBalancerWarning) && (
```

Comment:
but we've typically used double bang for coercing to a boolean type in the rest of the codebase, i.e., `!!loadBalancerWarning`

this isn't actually a preference of ours, but it would be nice to be consistent code-style-wise

Reply:
Sure I can do that. I have no strong preference either, just a habit of using `Boolean(...)` from previous Flow rules.

Reply:
There are days where I really miss Flow's magical `Boolean` inference powers...